### PR TITLE
Rename Gradle Enterprise to Develocity

### PR DESCRIPTION
**Pull request by welandaz**

### CODE REVIEW

**Review by alextu — APPROVED** [review]
LGTM, once merged smoke tests will need to be fixed to point to the new templates.

File: README.md [hunk 1]
```diff
@@ -17,46 +17,46 @@ The templates can also be configured to ad-hoc connect Gradle and Maven builds t
...
   before_script:
-    - !reference [.injectGradleEnterpriseForGradle]
+    - !reference [.injectDevelocityForGradle]
   script:
-    - ./gradlew check -I $GRADLE_ENTERPRISE_INIT_SCRIPT_PATH # Will publish a build scan to https://gradle-enterprise.mycompany.com
+    - ./gradlew check -I $DEVELOCITY_INIT_SCRIPT_PATH # Will publish a build scan to https://develocity.mycompany.com
 ```
-The `.injectGradleEnterpriseForGradle` creates an init script with the instrumentation logic and exports the path as `$GRADLE_ENTERPRISE_INIT_SCRIPT_PATH` environment variable.
-For all other options see `inputs` section in [gradle-enterprise-gradle.yml](gradle-enterprise-gradle.yml).
+The `.injectDevelocityForGradle` creates an init script with the instrumentation logic and exports the path as `$GRADLE_ENTERPRISE_INIT_SCRIPT_PATH` environment variable.
```
Comment (guylabs) [inline README.md]:
```suggestion
The `.injectDevelocityForGradle` creates an init script with the instrumentation logic and exports the path as `$DEVELOCITY_INIT_SCRIPT_PATH` environment variable.
```

**Review by guylabs — APPROVED** [review]
Please ensure that all build agents and CCUD's are updated to the latest version.

File: README.md (hunk 1 above)
Comment (welandaz) [reply README.md]: Done

**Pull request merged**